kubernetes_deploy/staging/deployment.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-fala
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: laa-fala-app
+  template:
+    metadata:
+      labels:
+        app: laa-fala-app
+    spec:
+      containers:
+      - image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-fala:master
+        name: app
+        ports:
+        - containerPort: 80
+          name: http
+        env:
+        - name: LAALAA_API_HOST
+          value: https://staging.laalaa.dsd.io
+        - name: DEBUG_STATIC
+          value: "True"
+        - name: DEBUG
```

Comment:
Do we need `DEBUG` too?

Reply:
No, but thought it might be helpful in our only non-prod environment.

What do you think?

Reply:
I'm unsure what effects it has and our other applications don't use it. I think it might be better to turn it on consistently once we know we need it. What do you think?

Reply:
Sure.